godot_project/editor/controls/menu_bar/menu_virtual_objects/menu_virtual_objects.gd

```diff
@@ -68,3 +69,7 @@ func _on_id_pressed(in_id: int) -> void:
 			workspace_context.create_object_parameters.set_create_mode_type(CreateObjectParameters.CreateModeType.CREATE_PARTICLE_EMITTERS)
 			MolecularEditorContext.request_workspace_docker_focus(CreateDocker.UNIQUE_DOCKER_NAME, &"Particle Emitters")
 			request_hide.emit()
+		ID_REFERENCE_SHAPES:
+			workspace_context.create_object_parameters.set_create_mode_type(CreateObjectParameters.CreateModeType.CREATE_SHAPES)
+			MolecularEditorContext.request_workspace_docker_focus(CreateDocker.UNIQUE_DOCKER_NAME, &"Reference Shape Parameters")
+			request_hide.emit()
```

Comment:
This was not part of the request and should be removed.

Reference Shapes already exists in the Create menu as a submenu of Create menu.
Even if a change is needed, that would be moving the shapes menu to the virtual objects submenu.

<img width="409" height="358" alt="image" src="https://github.com/user-attachments/assets/ce3dc3ea-c302-4034-8a8c-92a0a7234e1e" />